code/datums/uplink_item.dm

```diff
@@ -215,7 +257,7 @@ var/list/uplink_items = list()
 	name = "Viscerator Delivery Grenade"
 	desc = "A unique grenade that deploys a swarm of viscerators upon activation, which will chase down and shred any non-operatives in the area."
 	item = /obj/item/weapon/grenade/spawnergrenade/manhacks
-	cost = 8
+	cost = 5
```

Comment:
Reeeally not sure about this one. Viscerators are already incredible value for the price, but I'm willing to give it a try at 5TC and see how it goes.